tests/codemods/test_flask_send_file_path_parameterization.py

```diff
@@ -0,0 +1,178 @@
+
+    def test_indirect_path(self, tmpdir):
+        input_code = """\
+        from flask import Flask, send_file
+        from pathlib import Path
+
+        app = Flask(__name__)
+
+        @app.route("/uploads/<path:name>")
+        def download_file(name):
+            path = Path(f'path/to/{name}.txt')
+            return send_file(path)
+        """
+        expected = """\
+        from flask import Flask
+        from pathlib import Path
+        import flask
+
+        app = Flask(__name__)
+
+        @app.route("/uploads/<path:name>")
+        def download_file(name):
+            path = Path(f'path/to/{name}.txt')
+            return flask.send_from_directory(path.parent, path.name)
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(expected))
+        assert len(self.file_context.codemod_changes) == 1
+
+    def test_indirect_path_alias(self, tmpdir):
+        input_code = """\
+        from flask import Flask, send_file as send
+        from pathlib import Path
+
+        app = Flask(__name__)
+
+        @app.route("/uploads/<path:name>")
+        def download_file(name):
+            path = Path(f'path/to/{name}.txt')
+            return send(path)
+        """
+        expected = """\
+        from flask import Flask
+        from pathlib import Path
+        import flask
+
+        app = Flask(__name__)
+
+        @app.route("/uploads/<path:name>")
+        def download_file(name):
+            path = Path(f'path/to/{name}.txt')
+            return flask.send_from_directory(path.parent, path.name)
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(expected))
+        assert len(self.file_context.codemod_changes) == 1
+
+    def test_indirect_string(self, tmpdir):
+        input_code = """\
+        from flask import Flask, send_file
+
+        app = Flask(__name__)
+
+        @app.route("/uploads/<path:name>")
+        def download_file(name):
+            path = f'path/to/{name}.txt'
+            return send_file(path)
+        """
+        expected = """\
+        from flask import Flask
+        import flask
+        from pathlib import Path
+
+        app = Flask(__name__)
+
+        @app.route("/uploads/<path:name>")
+        def download_file(name):
+            path = f'path/to/{name}.txt'
+            return flask.send_from_directory((p := Path(path)).parent, p.name)
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(expected))
+        assert len(self.file_context.codemod_changes) == 1
+
+    def test_unknown_type(self, tmpdir):
+        input_code = """\
+        from flask import Flask, send_file
+
+        app = Flask(__name__)
+
+        @app.route("/uploads/<path:name>")
+        def download_file(name):
+            return send_file(name)
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(input_code))
+        assert len(self.file_context.codemod_changes) == 0
```

Comment:
not seeing a basic param of just a string without interpolation, like `send_file('path')`